.circleci/config.yml

```diff
@@ -32,6 +32,16 @@ jobs:
           echo 'export AZURE_AUTH_LOCATION=${outputPathCredsfile}' >> $BASH_ENV
       - run: |
           echo 'export KUBECONFIG=${outputPathKubeConfigFile}' >> $BASH_ENV
+      - run: |
```

Comment:
Not sure what is the best way to add the `nomad` binary to CircleCI. The binary is used for testing the provider. Just adding this as a step in CircleCI workflow.

Reply:
cc @robbiezhang

Reply:
give it a name "Install nomad"

Reply:
BTW, prefer to add it right before the "Tests" step